src/camvidlog2/yoloe/cli.py

```
@@ -0,0 +1,65 @@
+import typer
+
+from camvidlog2.common.data import load as data_load
+from camvidlog2.vid import generate_frames_cv2
+from camvidlog2.yoloe.ai import generate_tracked_bboxes
+from camvidlog2.yoloe.data import create
+
+app = typer.Typer()
+
+
+@app.command()
+def load(
```

Ruff (0.11.9)

Missing return type annotation for public function `load`

Add return type annotation: `None`

(ANN201)